### ISSUE TITLE
[4.0] pre-update check [a11y] and more

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#33691]
## Accessibility
Toggling a div is completely not accessible. You can not do this with anything other than a mouse. 
![image](https://user-images.githubusercontent.com/1296369/117570811-d0048300-b0c3-11eb-8797-bbf687e17320.png)

## Missing float-end 
![image](https://user-images.githubusercontent.com/1296369/117570869-10fc9780-b0c4-11eb-8dcd-02140ccead18.png)

## Its not more detail its all detail - if you were lucky enough to even notice that
![image](https://user-images.githubusercontent.com/1296369/117570900-32f61a00-b0c4-11eb-9415-6c263b4e0fe2.png)

## Inline styles should not be used
![image](https://user-images.githubusercontent.com/1296369/117570967-781a4c00-b0c4-11eb-9d04-f0c006882c11.png)

**Comment by hans2103** [comment]
@brianteeman I can find the code where this is set. But how do I reproduce the issue?

**Comment by brianteeman** [comment]
you need to trick joomla into thinking there is an update available.

what I did was to set the update url to custom and then used the url from the most recent pull request

**Comment by ceford** [comment]
Layout needs attention too. PHP Warning message is outside its box:
![screen shot 2021-06-16 at 09 09 20](https://issues.joomla.org/uploads/1/c38b739846774ddcf3fbbaf38e918daf.png)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33691">issues.joomla.org/tracker/joomla-cms/33691</a>.</sub>

**Comment by brianteeman** [comment]
closing as #34754 makes this redundant/resolved
